BEIMA.Backend.FT/DeviceFT.cs

```diff
@@ -82,25 +90,38 @@ public async Task OneTimeSetUp()
         public async Task SetUp()
         {
             // Delete all the devices in the database
-            var deviceList = await TestClient.GetDeviceList();
+            var deviceList = await TestClient.GetDeviceList(_authToken);
             foreach (var device in deviceList)
             {
                 if (device?.Id is not null)
                 {
-                    await TestClient.DeleteDevice(device.Id);
+                    await TestClient.DeleteDevice(device.Id, _authToken);
                 }
             }
         }
 
         [TestCase("xxx")]
         [TestCase("1234")]
         [TestCase("1234567890abcdef1234567x")]
```

Comment:
We might as well include some test cases with some validly formatted id's here as well.

Reply:
Went ahead and added a unauthorized test for every normal test